### DISCUSSION TITLE
Screen sleep format

### DISCUSSION THREAD

**Discussion by mhmd-cli** [i3/i3#5904] [Q&A]
## I'm submitting a…

<pre>
[x] Bug
[x] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
 While watching videos specifically ones that are outside of youtube using the Plyr or VideoJS video players, my screen shuts down during the player playing videos, even though it is in full screen, while in youtube for example, that doesn't happen.

## Impact
 Fault with usability of the software.
<pre>
[ ] This feature requires new configuration and/or commands
</pre>

## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
i3 version: i3 version 4.23 (2023-10-29) © 2009 Michael Stapelberg and contributors
</pre>

<pre>
- Linux Distribution & Version: OS: Arch Linux x86_64, Kernel: 6.7.2-arch1-2
- Are you using a compositor (e.g., xcompmgr or compton): I have compton/picom installed, but its because I recently cloned https://github.com/Axarva/dotfiles-2.0 which has it as one of its dependencies I'm assuming. But for i3, definitely didn't configure it to be used in my setup, I simply have the default settings.
</pre>

**Comment by okraits** [comment]
This has nothing to do with i3. It only manages X11 windows.

It seems that the youtube player generates some activity so that power management doesn't come into play. And the other players don't seem to do that. You need to adapt the power management settings for your screen.

Read here:

https://wiki.archlinux.org/title/Display_Power_Management_Signaling

**Comment by mhmd-cli** [comment]
The thing is that I also used gnome in my same setup and it worked just fine with the same players without shutting down. That is fair enough though to simply extend the screen wait time before shutdown, but ig its just extra features at this point, thanks for the quick help and if the other thing can get solved, that would be great.

**Reply by orestisfl** [reply]
i3 does not participate in the interaction between video players and dpms (Display Power Management Signaling), so it's not something we can or should interfere with.

That said, https://videojs.com/city perfectly inhibits the screensaver for me, tested with `xset s 1 1`
